ajax/libs/js-polyfills/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "js-polyfills",
+  "filename": "polyfill.min.js",
+  "author": "Joshua Bell <inexorabletash@gmail.com>",
+  "version": "0.1.21",
+  "description": "Collection of Web polyfills.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/inexorabletash/polyfill.git"
+  },
+  "keywords": [
+    "polyfill",
+    "standards"
+  ],
+  "license": "Public Domain",
```

Comment:
@PeterDaveHello 
I don't see this in [license](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json).
Is this correct?